.github/workflows/owlbot_sha256.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+on:
+  workflow_dispatch:
+  schedule:
+  # every hour
+  - cron: '0 * * * *'
+name: readme
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'googleapis'
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.7'
+        architecture: 'x64'
+    - run: python3 -m pip install --require-hashes -r .github/requirements.txt
+    - run: python3 generate-readme.py
+    - name: Replace sha256 value in OwlBot to the latest value
+      shell: bash
+      run: |
+        sha256=$(gcloud container images describe gcr.io/cloud-devrel-public-resources/owlbot-java --format 'value(image_summary.digest)')
```

Comment:
Ubuntu latest should have gcloud command https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md

Reply:
do you mean `Google Cloud SDK 421.0.0` in this [section](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#cli-tools)?

Reply:
Yes